synapseclient/remote_file_storage_wrappers.py

```diff
@@ -137,7 +137,11 @@ def upload_file(filepath, url, username=None, password=None):
 
         parsedURL = SFTPWrapper._parse_for_sftp(url)
 
-        with pysftp.Connection(parsedURL.hostname, username=username, password=password) as sftp:
+        # ignore hostkey
```

Comment:
in .travis.yml update this chunk of code to include the new address:
```
addons:
  #This is used for SFTP integration tests
ssh_known_hosts: ec2-54-212-85-156.us-west-2.compute.amazonaws.com
```
in appveyor.yml:
```
before_test:
- IF NOT DEFINED APPVEYOR_PULL_REQUEST_NUMBER ( ssh-keyscan -t ecdsa ec2-54-212-85-156.us-west-2.compute.amazonaws.com >> %userprofile%\\.ssh\\known_hosts )
```